lesson4/hometask_frameq.rb

```diff
@@ -1,16 +1,15 @@
 class LazyHash < Hash
   def method_missing(name)
     key = name.to_s
```

Comment:
It's not enough. Just try to add symbol key into a hash and call it as a method.
See,
```ruby
a[:symbol_key] = 'foo'
p a.symbol_key #=> `method_missing': undefined method `symbol_key' for {:symbol_key=>1}:LazyHash (NoMethodError)
```